croppie.js

```diff
@@ -1059,7 +1276,28 @@
         _updateOverlay.call(self);
     }
 
-    function _updateZoomLimits (initial) {
+    function getBounds(element) {
+        //How much do I miss babel? Let me list the ways.
+        var boundingClientRect = element.getBoundingClientRect()
+        return ["bottom", "height", "left", "right", "top", "width", "x", "y"].reduce(function (copy, key) {
+            return (copy[key] = boundingClientRect[key]) && copy;
+        }, {});
+
+    }
+
+    /**
+     * Get the elements client rect without border width because that causes zoom to be wrong
+     * @param element to get the client rect for
+     */
+    function getInnerDimensions(element) {
+        var copy = getBounds(element);
+        var computedStyles = window.getComputedStyle(element);

```

Comment:
probably won't affect our usecases but if element has a transform that affects width/height (like 'scale') the getComputedStyle won't reflect this. Probably not an issue?

Reply:
Ya, the library just straight up doesn't work on those things so 🍌